pyked/chemked.py

```diff
@@ -608,12 +610,20 @@ def __init__(self, properties):
                 setattr(self, prop.replace('-', '_'), None)
 
         self.composition_type = properties['composition']['kind']
-        composition = deepcopy(properties['composition']['species'])
-
+        # composition = deepcopy(properties['composition']['species'])
+        composition = []
+        for spec in properties['composition']['species']:
+            species_name = spec['species-name']
+            amount = spec['amount']
+            InChI = spec.get('InChI')
+            SMILES = spec.get('SMILES')
+            atomic_composition = spec.get('atomic-composition')
+            composition.append(Composition(species_name=species_name, InChI=InChI, SMILES=SMILES, atomic_composition=atomic_composition, amount=amount))
```

Comment:
Try to make sure the line length is less than 100 characters (I can't tell if this is or not)